### ISSUE TITLE
Tidy up JS

### ISSUE THREAD

**Issue by Android63** [NHSLeadership/nightingale-moodle#80]
Load different types of JS as defined in Nightingale https://github.com/NHSLeadership/nightingale/issues/191

**Comment by ishanivardhan** [comment]
@Android63 - Can you elaborate more on what needs to be done in this issue? I can see that Harry added JS snippets and included JS files at a couples of places in above referenced issue. Do we need to simply mimic that in our code?

**Comment by Android63** [comment]
Yes @ishanivardhan. We need to follow Harry's example so our JS gets loaded at the optimum moments.
